File: pythonexercicios/mes_se.py
"""
    Problema: Ler para uma variável INTEIRA um número de 1 a 12
    e mostrar o nome do mês correspondente.
    Caso o mês não existir, mostrar essa informação.

    Esse programa lê um número inteiro de 1 a 12, verifica qual o número
    escolido e mostra o mês correspondente, caso o mês não exista mostra
    essa informação.

    Autor: José Brenon - 09/08/2023
"""
mes = int(input("Digite um número de 1 a 12 para saber o mês correspondente: "))

if mes == 1:
    print("JANEIRO")
elif mes == 2:
    print("FEVEREIRO")
elif mes == 3:
    print("MARÇO")
elif mes == 4:
    print("ABRIL")
elif mes == 5:
    print("MAIO")
elif mes == 6:
    print("JUNHO")
elif mes == 7:
    print("JULHO")
elif mes == 8:
    print("AGOSTO")
elif mes == 9:
    print("SETEMBRO")
elif mes == 10:
    print("OUTUBRO")
elif mes == 11:
    print("NOVENBRO")
elif mes == 12:
    print("DEZEMBRO")
else:
    print("Esse mês não existe!")

print("{:.^21}".format("FIM"))

File: javaexercicios/mes_se.java
/*
    Problema: Ler para uma variável INTEIRA um número de 1 a 12
    e mostrar o nome do mês correspondente.
    Caso o mês não existir, mostrar essa informação.

    Esse programa lê um número inteiro de 1 a 12, verifica qual o número
    escolido e mostra o mês correspondente, caso o mês não exista mostra
    essa informação.

    Autor: José Brenon - 09/08/2023
*/
import javax.swing.JOptionPane;
public class mes_se {
    public static void main(String[] args) {
        int mes = Integer.parseInt(JOptionPane.showInputDialog("Digite o mês de 1 a 12: "));

        if (mes == 1) JOptionPane.showMessageDialog(null, "JANEIRO");
        else if (mes == 2) JOptionPane.showMessageDialog(null, "FEVEREIRO");
        else if (mes == 3) JOptionPane.showMessageDialog(null, "MARÇO");
        else if (mes == 4) JOptionPane.showMessageDialog(null, "ABRIL");
        else if (mes == 5) JOptionPane.showMessageDialog(null, "MAIO");
        else if (mes == 6) JOptionPane.showMessageDialog(null, "JUNHO");
        else if (mes == 7) JOptionPane.showMessageDialog(null, "JULHO");
        else if (mes == 8) JOptionPane.showMessageDialog(null, "AGOSTO");
        else if (mes == 9) JOptionPane.showMessageDialog(null, "SETEMBRO");
        else if (mes == 10) JOptionPane.showMessageDialog(null, "OUTUBRO");
        else if (mes == 11) JOptionPane.showMessageDialog(null, "NOVEMBRO");
        else if (mes == 12) JOptionPane.showMessageDialog(null, "DEZEMBRO");
        else JOptionPane.showMessageDialog(null, "Esse mês não existe");
        System.out.println("......FIM.......");
    }
}
